services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/events.py

```diff
@@ -501,7 +511,16 @@ async def action(cls, app: FastAPI, scheduler_data: SchedulerData) -> None:
                             dynamic_sidecar_endpoint,
                         )
                     )
-                await logged_gather(*tasks)
+
+                try:
+                    await logged_gather(*tasks)
+                except NodeportsDidNotFindNodeError:
```

Comment:
We should also do a bigger effort in defining exceptions (e.g. with some sort of hierarchy or code)
If for every error we define a convoluted name for an exception, the number of combinations will not only explode but will be impossible to handle. 
For instance, It would be nice to associate this type of exception with a node error, with a node id (e.g. the warning gets that info from the scheduler_data instead of the error itself ...)